am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -39,6 +42,9 @@
 
 @SuppressWarnings({"PMD.ExcessiveImports", "PMD.TooManyMethods", "LineLength"})
 class FilterResourceIntegrationTest extends PreconfiguredIntegrationBaseTest {
+
+    private static final String NAME_ATTRIBUTE = "/name";
+    private static final String CITY_ATTRIBUTE = "/address/city";
```

Comment:
change to generic rootLevelAttribute and nestedAttribute for consistency as discussed

Reply:
As discussed,will randomize this.

Reply:
done randomization